### PR TITLE
Added RTL swipe to close the drawer

### PR DESCRIPTION
**Pull request by goldoraf**
Not much to add, besides the fact that the `overflow-y : scroll` bug on Chrome made things more... interesting ;)

### CODE REVIEW

**Comment by goldoraf** [comment]
[(2) 📱 Plus moyen de fermer le menu sandwich de Hambourg](https://trello.com/c/0jAwWSVq/821-2-%F0%9F%93%B1-plus-moyen-de-fermer-le-menu-sandwich-de-hambourg)

File: src/components/Drawer.jsx [hunk 1]
```diff
@@ -35,6 +51,88 @@ class Drawer extends Component {
...
+      this.applyTransformation(currentGestureProgress)
+    })
+
+    this.gesturesHandler.on('panend', e => {
+      if (this.state.isScrolling) {
+        this.setState({isScrolling: false})
+        return
+      }
+      // Dismiss the menu if the swipe pan was bigger than the treshold,
+      // or if it was a fast, downward gesture
```
Comment (y-lohse) [inline src/components/Drawer.jsx]: s/downward/leftward

**Comment by goldoraf** [comment]
@y-lohse the problem is that if we only listen to horizontal movements, when the user attemps to scroll, a single erroneous `panleft` (with a big deltaX) is emitted and the drawer is gets stuck with a weird width. That's the only fix that could possibly work.

**Comment by y-lohse** [comment]
Alright, works for me 👍

**Comment by CPatchane** [comment]
Thanks @goldoraf and @y-lohse 👍

**Pull request merged**